scripts/pylib/twister/twisterlib.py

```diff
@@ -2809,13 +2840,17 @@ def get_all_tests(self):
 
     @staticmethod
     def get_toolchain():
-        toolchain = os.environ.get("ZEPHYR_TOOLCHAIN_VARIANT", None)
+        toolchain_script = ZEPHYR_BASE + "/cmake/verify-toolchain.cmake"
```

Comment:
Does twister run on windows these days? If so, you might want to use pathlib instead of raw string manipulation, to avoid possibly intermixing backslash and forward-slash path elements; that causes issues, right?

Reply:
you're absolutely right.
Even though we can't run twister on windows today, afaik, then we should still make proper python code.

To used to CMake always handling `/` properly, which gives me bad habits when jumping from CMake to python 😞 

Will create followup commit.